regression/scripts/segments/transition_segment_test.py

```diff
@@ -240,6 +240,50 @@ def mission_setup(analyses,vehicle):
     # add to misison
     mission.append_segment(segment)
     
+    # --------------------------------------------------------------------------
+    #   Segment 2a: Transition Segment: Linear Speed, Linear Climb
+    # --------------------------------------------------------------------------
+    # Use original transition segment, converge on rotor y-axis rotation and throttle
+    segment                                             = Segments.Transition.Constant_Acceleration_Constant_Angle_Linear_Climb(base_segment)
+    segment.tag                                         = "Transition_2a"
+    segment.analyses.extend( analyses.transition_1 )
+    segment.altitude_start                              = 40.0 * Units.ft
+    segment.altitude_end                                = 100.0 * Units.ft
+    segment.acceleration                                = 0.5  * Units['m/s/s']
+    segment.climb_angle                                 = 7. * Units.deg
+    segment.pitch_initial                               = 3.6  * Units.degrees  
+    segment.pitch_final                                 = 4.0  * Units.degrees   
+    segment.state.unknowns.throttle                     = 0.9  * ones_row(1)
+    segment.process.iterate.unknowns.mission            = SUAVE.Methods.skip
```

Comment:
if all tiltrotor transitions don't use the missions unknowns it can be added to the add_tiltrotor_transition_unknowns_and_residuals_to_segment

Reply:
updated